amundsen_application/static/css/_typography-default.scss

```diff
@@ -3,6 +3,150 @@
 
 @import 'variables';
 
+// New Typography styles based on LPL
+// Use these styles going forward
+
+// Placeholder selectors
+// Ref: http://thesassway.com/intermediate/understanding-placeholder-selectors
+
+%lpl-headline-w1 {
```

Comment:
I don't think we need to reference `lpl` here since it doesn't mean anything to the open source community, and the open source version of Amundsen isn't expected to keep in lock step with LPL since designers will be expected to collaborate. So I think a more generic name would be great even if we start of in lock step with LPL.

Reply:
Sounds good!
How about `text-*` ?